### DISCUSSION TITLE
Are the benchmarks up to date?

### DISCUSSION THREAD

**Discussion by rafaell-lycan** [sindresorhus/got#2176] [Q&A] [answered]
#### What would you like to discuss?

Hello 👋 

I'd like to ask if the Benchmarks are up to date mainly because we are considering changing `node-fetch` in favor of `got` at my company, but we gotta know if the benchmarks for promises are actually faster or at least the same as `node-fetch`.

I was trying to run the project locally `npm install & npm run build && node ./dist/benchmark/index.js` but it didn't work.

Is there any place were we can check `got` compared with the latest version of `node-fetch`, `axios`, etc?

Thanks.

#### Checklist

- [x] I have read the documentation.

**Comment by szmarczak** [comment, marked as answer]
> I was trying to run the project locally npm install & npm run build && node ./dist/benchmark/index.js but it didn't work.

You have to run the server first and change the port to `8080` since the benchmark was ran on H2O server to make it not use so much CPU, to give more CPU for the clients.

> I'd like to ask if the Benchmarks are up to date

They haven't been updated in 15 months, but I don't think much changed.

What's the reason for the change?
